src/components/Logo/__tests__/Logo.test.js

```diff
@@ -36,9 +35,6 @@ describe('components/Logo', () => {
     expect(
       container.querySelector(`a[href="${configuration.links.home}"]`)
     ).toBeInTheDocument()
-    expect(
-      container.querySelector(`img[src="${configuration.logo.url}"]`)
```

Comment:
Shouldn't we keep these tests now?

Reply:
the url definition on line 18 should be removed, but we could put in the fixed logo url..

Reply:
I'll put the tests back. Good catch.